chainguard-dev/edu
# Add dynamic vulnerability and image comparison pages

## Type of change
platform

### What should this PR do?
Adds two new page types, image comparison and vulnerability comparison.

### Why are we making this change?
We have great data to show off!

### What are the acceptance criteria? 
Everything should render correctly, save for these two new page types since I think netlify won't like them.

### How should this PR be tested?
Check netlify, and if that doesn't work, staging should have a copy of these same changes.
